harness/determined/experimental/client.py

```diff
@@ -34,27 +34,37 @@ def login(
     noverify: bool = False,
 ) -> None:
     """
-    login will configure the default Determined() singleton used by all of the other functions in
-    the client module.
+    ``login()`` will configure the default Determined() singleton used by all of the other functions
+    in the client module.
 
-    It is often unnecessary to call login().  If you have configured your environment so that
+    It is often unnecessary to call ``login()``.  If you have configured your environment so that
     the Determined CLI works without any extra arguments or environment variables, you should not
-    have to call login at all.
+    have to call ``login()`` at all.
 
-    If you do need to call login(), it must be called before any calling any other functions from
-    this module, otherwise it will fail.
+    If you do need to call ``login()``, it must be called before any calling any other functions
+    from this module, otherwise it will fail.
 
-    If you have reason to connect to multiple masters, you should use explicit Determined() objects
-    instead.  Each explicit Determined() object accepts the same parameters as login(), and offers
-    the same functions as what are offered in this module.
+    If you have reason to connect to multiple masters, you should use explicit
+    :class:`~determined.experimental.Determined` objects instead.  Each explicit
+    :class:`~determined.experimental.Determined` object accepts the same parameters as
+    ``login()``, and offers the same functions as what are offered in this module.
 
     .. note::
 
        Try to avoid having your password in your python code.  If you are running on your local
-       machine, you should always be able to use ``det user login`` on the CLI, and login()
+       machine, you should always be able to use ``det user login`` on the CLI, and ``login()``
        will not need either a user or a password.  If you have ran ``det user login`` with multiple
        users (and you have not ran ``det user logout``), then you should be able to run
        ``login(user=...)`` for any of those users without putting your password in your code.
+
+    Arguments:
+        master (string, optional): The URL of the Determined master.
+            If this argument is not specified, the environment variables
+            DET_MASTER and DET_MASTER_ADDR will be checked for the master URL in that order.
+        user (string, optional): The Determined username used for
+            authentication. (default: ``determined``)
+        password (string, optional): The password associated with the Determined
+            user.
```

Comment:
can you remove the word `Determined`?  That sounds a bit like it wants the name for the user whose username is `determined`.